action.yml

```diff
@@ -2,6 +2,10 @@
 name: 'Should Run GH Action'
 description: 'A reusable GitHub action to determine if the user clicked a checkbox inn the PR text'
 author: 'Gleb Bahmutov'
+inputs:
+  checkbox_text:
+    description: The checkbox text to search
+    required: true
```

Comment:
required or has a default? I would make it have the default test "re-run the tests" to avoid making a breaking change

Reply:
Great idea! I'll make the change